prometheus-ksonnet/lib/prometheus.libsonnet

```diff
@@ -30,13 +30,15 @@
 
     local container = $.core.v1.container,
 
+    prometheus_config_file:: '/etc/prometheus/prometheus.yml',
+
     prometheus_container::
       local _config = self._config;
 
       container.new('prometheus', $._images.prometheus) +
       container.withPorts($.core.v1.containerPort.new('http-metrics', _config.prometheus_port)) +
       container.withArgs([
-        '--config.file=/etc/prometheus/prometheus.yml',
```

Comment:
Would be good to turn these into a map as we do in cortex: https://github.com/grafana/cortex-jsonnet/blob/master/cortex/distributor.libsonnet#L45

Reply:
I like that, but.... it changes the order of the args, causing a redeploy. Would cause a re-deploy of *both* the old and the new prometheus servers! And given that redeploying prometheus is what we're trying to make smoother, this isn't the time to make this change, I'm afraid.